### DISCUSSION TITLE
Timelapse Preview never shows anything

### DISCUSSION THREAD

**Discussion by MatCat** [FormerLurker/Octolapse#919] [Q&A]
Not sure what is wrong, but I just setup a new octopi with latest version, running octolapse, it generates a timelapse video just fine of a print, however the preview area always just shows "No snapshots have been taken with the current camera.  A preview of your timelapse will start to appear here as snapshots are taken by Octolapse.", I can see the snapshot count count up as it takes them, and even see a little loading like animation for a second or 2 when it takes one in the preview area, but it never shows.

**Comment by MatCat** [comment]
After looking at F12, I noticed this error:

`Failed to load resource: the server responded with a status of 404 (Not Found)      getSnapshot:1`

**Comment by trunzoc** [comment]
Did you ever resolve this?  I have the same issue.
